### DISCUSSION TITLE
Should/Can i Remove the component that i won't be using?

### DISCUSSION THREAD

**Discussion by Prakash4844** [McShelby/hugo-theme-relearn#458] [Q&A] [answered]
First of all i am new to hugo, less than 4 days new. So i was thinking suppose i won't be using mathjax in my project, can i remove it without breaking something? also should i even remove it? will it affect the performance?

**Comment by McShelby** [comment, marked as answer]
Welcome to the club.

I haven't tested this and **won't recommend** it. It's difficult to do, as those components (MathJax, Mermaid and RapiDoc) are referenced in multiple file throughout the themes. Deleting all those references aswell will make it difficult for your once you want to update the theme to a newer version.

Components / Shortcodes you aren't using have almost no overhead in runtime performance when creating the page and do **not** result in additional code / files to download for your HTML pages.

**Reply by Prakash4844** [reply]
Okay, thanks for clarifying.
